package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@temporalio/ui",
-  "version": "2.1.53",
+  "version": "2.1.54",
```

Comment:
[Looks](https://www.npmjs.com/package/@temporalio/ui) like we might want `2.1.55` here.